source/extensions/common/wasm/context.cc

```diff
@@ -980,7 +982,13 @@ WasmResult Context::grpcCall(absl::string_view grpc_service, absl::string_view s
                              uint32_t* token_ptr) {
   GrpcService service_proto;
   if (!service_proto.ParseFromArray(grpc_service.data(), grpc_service.size())) {
-    return WasmResult::ParseFailure;
```

Comment:
Could we support using existing cluster here, by providing only the cluster name, and not a proto service (regardless of whether it's binary or YAML)?